src/docs/development/ios-14.md

```diff
@@ -39,7 +39,8 @@ functionalities such as hot-reload and DevTools.
 ![Screenshot of "allow network connections" dialog]({% asset development/device-connect.png @path %})
 
 This affects debug and profile builds only and won't
-appear in release builds. 
+appear in release builds. The permission can also be allowed
+by enabling `Settings > Privacy > Local Network > Your App`.
```

Comment:
I think Shams wants these UI references to be bolded rather than code formatted

Reply:
You're right, I forgot.